### DISCUSSION TITLE
Url Scheme Problems

### DISCUSSION THREAD

**Discussion by Goojoe** [vuepress-theme-hope/vuepress-theme-hope#2554] [Q&A] [answered]
I want to by `vscode://file/D:/Github/vue-docs` this url scheme to quickly open the vscode, but I add links to the Home pages link is
```
http://localhost:8080/vscode://file/D:/Github/vue-docs
```

**Comment by Mister-Hope** [comment, marked as answer]
Not sure what you are asking, you'd better describe with more details explaining what you want and what you are doing. E.g.: you want a a hotdog to eat and you enter a bookshop and ask the waiter to provide one.

For a normal docs site, most situations they don't need custom scheme url link as users may failed to open them because lacking of software supporting them. If you do have a need to provide a button for all visitors to open vscode and choose d disk, you may need to write a button by your own. Users usually do not have that folder and do not have vscode installed, and also they may not use a pc having windows os installed.
